features/karaf/openhab-tp/src/main/feature/feature.xml

```diff
@@ -33,6 +33,9 @@
     <!-- Gson -->
     <bundle dependency="true">mvn:com.google.code.gson/gson/2.3.1</bundle>
 
+    <!-- JBBP -->
+    <bundle dependency="true">mvn:com.igormaznitsa/jbbp/1.4.1</bundle>
```

Comment:
Can you create a separate feature for that bunde (e.g. like `openhab.tp-commons-net`)?
The separate bin2json feature in openhab-core feature repo could then depend on the tp (by adding a "dependency=true" dependency to that TP feature)